CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR})
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/cmake)
 
 # options shared by all targets
+option(NO_GIT "Don't use git revision timestamp" OFF)
```

Comment:
why the negative, I would prefer:
`option(GIT_TIMESTAMP "Use git revision timestamp" ON)`